src/frontends/ir/tests/test_model_with_missing_weights.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" ?>
```

Comment:
Please remove all models from file systems, we can create files in the runtime

Reply:
done